### DISCUSSION TITLE
How did you guys detach the fork

### DISCUSSION THREAD

**Discussion by JPcode05** [tenacityteam/tenacity-legacy#246] [Q&A] [answered]
This GitHub repo used to say it was a fork of Audacity, and now it's not. How were you guys able to detach the repository? I am trying to do it for a [fork of GNOME Shell 3.38](https://github.com/JPcode05/gnome-shell).

**Comment by SubOptimal** [comment, marked as answer]
From the [documentation](https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/managing-contribution-graphs-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#commit-was-made-in-a-fork)

> To detach the fork and turn it into a standalone repository on GitHub, contact GitHub Support or GitHub Premium Support.

On the support [request](https://support.github.com/request) page there is a [direct link](https://support.github.com/contact?tags=rr-forks) for such a request.

![grafik](https://user-images.githubusercontent.com/653288/125400621-69952d00-e3b2-11eb-8707-bc128f32171e.png)

**Reply by fossdd** [reply]
Also see: https://github.com/tenacityteam/tenacity/issues/38
